lib/util.js

```diff
@@ -22,6 +22,18 @@ module.exports.injectPartial = function(base_html, { options, html, priority, lo
 
   }
 
+  /*
+  *
+  *   Option: replaceElement
+  *   Behavior: Replaces location element instead of becoming child of element
+  *
+  * */
+  if(options["replaceElement"] != null){
```

Comment:
a few style update requests

can you update the comment block to look like:
```
/**
 *
 */
```

i think i would prefer to see `options.replaceElement` instead of `options["replaceElement"]`

we  could also probably strictly check that replaceElement is `true` given it seems like it should be a boolean, so:
```
if ( options.replaceElement === true )
```

Reply:
Done, and done!